test/e2e_node/gpu_device_plugin.go

```diff
@@ -36,6 +37,7 @@ import (
 const (
 	devicePluginFeatureGate = "DevicePlugins=true"
 	testPodNamePrefix       = "nvidia-gpu-"
+	nonDeviceResource       = "test.com/testResource"
```

Comment:
Nit: there are some top-level and second-level domains reserved by IANA specifically for testing and docs, but `test.com` isn't one of them: see https://tools.ietf.org/html/rfc2606

Reply:
Thanks for the info! Changed to use example.com.